aws-dynamodb-point-in-time-recovery-disabled.yaml

```
@@ -0,0 +1,37 @@
+rules:
+- id: aws-dynamodb-point-in-time-recovery-disabled
+  patterns:
+  - pattern: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+      }
+  - pattern-not-inside: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+        point_in_time_recovery {
+          enabled = true
+        }
+        ...
+      }
+  message: Point-in-time recovery is not enabled for the DynamoDB table. DynamoDB
+    tables should be protected against accidental or malicious write/delete actions.
+    By enabling point-in-time-recovery you can restore to a known point in the event
+    of loss of data.
+  languages:
+  - hcl
+  severity: WARNING
```

impact and likelihood are low, that means this should be an INFO level rule
```suggestion
  severity: INFO
```